client/components/deposits-information/index.tsx

```diff
@@ -19,36 +19,41 @@ import {
 	getNextDepositLabelFormatted,
 } from 'deposits/utils';
 import InstantDepositButton from 'deposits/instant-deposits';
+import StandardDepositButton from 'wcpay/components/standard-deposit-button';
 import DepositsInformationBlock from './block';
 import { formatCurrency, formatCurrencyName } from 'utils/currency';
 import { useAllDepositsOverviews } from 'wcpay/data';
 
 import './style.scss';
 
-interface OverviewProps {
+type OverviewProps = {
```

Comment:
Ah, this is just personal preference. I always default to `type` over `interface` because they feel a little more predictable. I found this comment which sums up my thoughts:

>Also, an interface can be extended in several locations while a type has to be fully defined in one place. For me that's +100pts for types because when I can be sure by looking at a type def that I have the whole object, while interfaces definitions can be scattered all around, possibly in different files

Source: https://dev.to/saranshk/typescript-the-difference-between-interface-and-type-ol2#comment-1ekg4

Reply:
> an interface can be extended in several locations

Nice, so I guess it's a bit like `type` = `const`, `interface` = `let`.

Also, from the [TS docs/handbook](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces):

> Type aliases and interfaces are very similar, and in many cases **you can choose between them freely**, and TypeScript will tell you if it needs something to be the other kind of declaration

😝 This is why I haven't yet decided which to use, but I'll default to `type` from now on. Thanks for the tip!